### PR TITLE
Add XDEFI Wallet adapter

### PR DESCRIPTION
**Pull request by loxator**
Adding XDEFI Wallet to the Solana-Wallet-Adapter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
         "docs": "shx rm -rf docs && NODE_OPTIONS=--max_old_space_size=16000 typedoc && shx cp ./{.nojekyll,wallets.png} docs/",
         "deploy:docs": "pnpm run docs && gh-pages --dist docs --dotfiles",
         "example": "pnpm run --filter {packages/starter/example} export",
-        "deploy:example": "pnpm run example && gh-pages --dist packages/starter/example/out --dest example --dotfiles"
+        "deploy:example": "pnpm run example && gh-pages --dist packages/starter/example/out --dest example --dotfiles",
+        "tsc": "tsc --build --verbose tsconfig.all.json"
```
Comment (loxator) [inline package.json:34]: Added this as ReadMe mentions on step 3 to `pnpm run tsc` to build packages, but when running it gives out saying `Missing Script Error`

Comment (jordansexton) [reply package.json:34]: Ah, this is because the readme should be updated instead to call `pnpm run build:ts`.

Comment (loxator) [reply package.json:34]: Cool, i will revert this change then!

**Comment by jordansexton** [comment]
Looks good!

I also want to draw your attention to https://github.com/wallet-standard/wallet-standard

It's still WIP, so while I'll continue to merge adapter PRs for the next few weeks, we are quickly moving in the direction of no longer needing adapters for wallets that implement the standard.

**Comment by loxator** [comment]
@jordansexton , thanks for merging 🙌  Any idea when this package will be released? Thanks!

**Comment by jordansexton** [comment]
Done!

- @solana/wallet-adapter-xdefi@0.1.0
- @solana/wallet-adapter-wallets@0.18.8

**Pull request merged**